tests/testthat/test-which_grepl_linter.R

```diff
@@ -0,0 +1,27 @@
+test_that("which_grepl_linter skips allowed usages", {
+  # this _could_ be combined as p1|p2, but often it's cleaner to read this way
```

Comment:
Actually, with conjunction, it's not even doable since matches might overlap. The comment would be correct for alternation (OR) of the `grepl()` results.

Reply:
good catch